.circleci/config.yml

```
@@ -364,14 +379,10 @@ workflows:
           parallelism: 1
           matrix:
             parameters:
-              ruby-version:
-                - '3.1'
-                - '3.2'
-                - '3.3'
-              gem-version:
-                - rails-6.1.7
-                - rails-7.0.8
-                # https://github.com/doximity/super_spreader/issues/60
-                # - rails-7.1.3
+              ruby-version: ['3.1', '3.2', '3.3']
+              gem-version: ['rails-7.1.3', 'rails-7.2.2', 'rails-8.0.2']
+            exclude:
+              - ruby-version: '3.1'
+                gem-version: 'rails-8.0.2'
```

Ruby 3.1 is EOL: https://endoflife.date/ruby

Rather than excluding the 3.1/rails8 combo from the matrix, I'd probably just not include 3.1 at all.

You can also specify major.minor on the rails matrix and it'd install whatever the latest patch version is.

```suggestion
              ruby-version: ['3.2', '3.3', '3.4']
              gem-version: ['rails-7.1', 'rails-7.2', 'rails-8.0']
```